### PR TITLE
Right click menu

### PR DESCRIPTION
**Pull request by jjeffryes**
- adds a right click menu with
- undo
- redo
- copy
- paste
- paste and match style
- select all
- toggle fullscreen
- maximize
- unmaximize
- minimize

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -121,6 +130,115 @@ <h2 class="txt-center">
     $('.js-defaultServerIndexApp').on('click', function(){
       app.loadNewServer("http://localhost:18469/api/v1/");
     });
+
+    var appMenu = Menu.buildFromTemplate([
```
Comment (rmisio) [inline index.html]: Does this have to live in index.html? Our other menus live in main.js. If that files is getting too big, I'm sure we could break it up, but IMHO, plopping this in here seems odd.

Comment (jjeffryes) [inline index.html]: The examples on electron's documentation put it in index. It could probably move to js/main.js.

Comment (rmisio) [inline index.html]: I meant main.js, not js/main.js. That's where the other menus are. I guess it's not that huge of a deal.

Comment (jjeffryes) [inline index.html]: I wasn't able to get it to recognize the right click on the BrowserWindow in the main main.js. I can take another look at it.

**Pull request merged**